README.md

```diff
@@ -10,6 +10,8 @@ A STARK-based virtual machine.
 
 **WARNING:** This project is in an alpha stage. It has not been audited and may contain bugs and security flaws. This implementation is NOT ready for production use.
 
+**WARNING:** For `no_std`, only the WASM environment is officially supported.
```

Comment:
*Nit:* I would be more precise here and specify the target triples, namely `wasm32-unknown-unknown` and `wasm32-wasip1` (we don't actively build for the latter, nor do we have reason to for now, but it is a trivial extension of the former that we should support if we support Wasm at all).

Reply:
fixed :)